app/assets/stylesheets/desktop/compose.scss

```diff
@@ -83,8 +83,8 @@
 .composer-popup {
   box-sizing: border-box;
   position: absolute;
-  width: calc(50% - 8px);
-  top: 8px; // .reply-to margin-top + .reply-area padding-top
+  width: calc(50% - 16px);
+  top: 19px; // grippie (11px) + .reply-area padding (8px)
```

Comment:
should we use some kind of variable here? that seems ultra brittle for the future

Reply:
Grippie's height is currently determined by its content and top+bottom padding, so it can't be easily expressed by a variable. We'd have to change its styling to have a static height instead, and then use that here.

Or, change the DOM structure of the composer so that the main area is wrapped in a div (with position: relative) that's separate from the grippie.

Either way, those are bigger changes that will have to wait after the release. But definitely a good point!